### PR TITLE
feat: Move from assets in package to s3 bucket

### PR DESCRIPTION
**Pull request by iisakkirotko**

### CODE REVIEW

**Comment by iisakkirotko** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #462** <a href="https://app.graphite.dev/github/pr/widgetti/solara/462?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/widgetti/solara/462?utm_source=stack-comment).

File: solara/components/file_download.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def Page():
     def Page():
         with solara.FileDownload(data, "solara-download-2.txt"):
             solara.Markdown("Any text, or even an image")
-            solara.Image("https://solara.dev/static/public/beach.jpeg", width="200px")
+            solara.Image("https://dxhl76zpt6fap.cloudfront.net/public/beach.jpeg", width="200px")
```
Comment (maartenbreddels) [inline solara/components/file_download.py:61]:
```suggestion
            solara.Image("solara.dev/static/public/beach.jpeg", width="200px")
```

File: solara/lab/components/menu.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def ClickMenu(
 
     @solara.component
     def Page():
-        image_url = "/static/public/beach.jpeg"
+        image_url = "https://dxhl76zpt6fap.cloudfront.net/public/beach.jpeg"
```
Comment (maartenbreddels) [inline solara/lab/components/menu.py:39]:
```suggestion
        image_url = "https://solara.dev/static/public/beach.jpeg"
```

File: solara/lab/components/menu.py [hunk 3]
```diff
@@ -90,7 +90,7 @@ def ContextMenu(
...
```
Comment (maartenbreddels) [inline solara/lab/components/menu.py:93]:
```suggestion
        image_url = "https://solara.dev/static/public/beach.jpeg"
```

**Review by maartenbreddels — CHANGES_REQUESTED** [review]

File: solara/components/file_download.py [hunk 4]
```diff
@@ -58,7 +58,7 @@ def Page():
     def Page():
         with solara.FileDownload(data, "solara-download-2.txt"):
             solara.Markdown("Any text, or even an image")
-            solara.Image("https://solara.dev/static/public/beach.jpeg", width="200px")
+            solara.Image("solara.dev/static/public/beach.jpeg", width="200px")
```
Comment (maartenbreddels) [inline solara/components/file_download.py:61]:
```suggestion
            solara.Image("https://solara.dev/static/public/beach.jpeg", width="200px")
```

File: solara/lab/components/menu.py [hunk 5]
```diff
@@ -36,7 +36,7 @@ def ClickMenu(
 
     @solara.component
     def Page():
-        image_url = "/static/public/beach.jpeg"
+        image_url = "https://solara.dev/static/public/beach.jpeg"
```
Comment (maartenbreddels) [inline solara/lab/components/menu.py:39]:
```suggestion
        image_url = "/static/public/beach.jpeg"
```

File: solara/lab/components/menu.py [hunk 6]
```diff
@@ -90,7 +90,7 @@ def ContextMenu(
...
```
Comment (maartenbreddels) [inline solara/lab/components/menu.py:93]:
```suggestion
        image_url = "/static/public/beach.jpeg"
```

**Review by maartenbreddels — APPROVED** [review]

**Pull request merged**